### ISSUE TITLE
Dropout

### ISSUE THREAD

**Issue by harristeague** [THUKey/pytorch-mobilenet-v3#3]
the paper mentioned training with dropout.  How would you train with dropout with this model?

**Comment by bhack** [comment]
@harristeague These impl have dropout 
https://github.com/leaderj1001/MobileNetV3-Pytorch
https://github.com/Randl/MobileNetV3-pytorch

This one it is directly mentioning @kuan-wang for this repo:
https://github.com/PengBoXiangShang/MobileNetV3_PyTorch#discussion-about-dropout-08

**Comment by kuan-wang** [comment]
Thanks for reminding, I'd added dropout in the codes.

**Comment by kuan-wang** [comment]
But in my experience, no dropout is the best for small model and my training settings are:
number of epochs: 150
learning rate schedule: cosine learning rate, initial lr=0.05
weight decay: 4e-5
remove dropout
